Userland/Utilities/man.cpp

```diff
@@ -116,7 +116,23 @@ ErrorOr<int> serenity_main(Main::Arguments arguments)
     auto buffer = file->read_all();
     auto source = String::copy(buffer);
 
-    const String title("SerenityOS manual");
+    String title;
+    if (*section == '1')
+        title = "SerenityOS User Programs Manual";
+    else if (*section == '2')
+        title = "SerenityOS System Calls Manual";
+    else if (*section == '3')
+        title = "SerenityOS Library Functions Manual";
+    else if (*section == '4')
+        title = "SerenityOS Special Files Manual";
+    else if (*section == '5')
+        title = "SerenityOS File Formats Manual";
+    else if (*section == '6')
+        title = "SerenityOS Games Manual";
+    else if (*section == '7')
+        title = "SerenityOS Miscellanea Manual";
+    else if (*section == '8')
+        title = "SerenityOS Sysadmin Tools Manual";
```

Comment:
I think a slightly neater solution would be to change the `sections` array above to an array of 
```c++
struct Section {
   char const * number;
   char const * name;
}
```
Then here you can just do:  `auto title = String::formatted("SerenityOS {} Manual", section.name)`

Reply:
I like that idea!